exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/GrpcStatusUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.internal;
+
+import com.google.protobuf.CodedInputStream;
+import java.io.IOException;
+
+/** Utilities for working with gRPC status without requiring dependencies on gRPC. */
+public final class GrpcStatusUtil {
+
+  /** Parses the message out of a serialized gRPC Status. */
+  public static String getStatusMessage(byte[] serializedStatus) throws IOException {
```

Comment:
Realized it's easy to parse manually to reduce dependency. The main reason for finding the issue in the first place was to evaluate feasibility of proto-lite, and a dependency on the google proto would block that.